neo/Network/P2P/LocalNode.cs

```diff
@@ -156,6 +157,49 @@ protected override void OnReceive(object message)
             }
         }
 
+        protected override void OnUdpMessage(IPEndPoint remote, ByteString data)
```

Comment:
Can we reuse `ProtocolHandler` to handle the udp messages?

Reply:
Check it, my idea is that udp will be used only for receive transaction in a faster way